e2e/workspace-activity.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { test, expect } from "@playwright/test";
```

Comment:
these are mainly smoke tests for the pull requests table. More thorough tests will come later once we have our whole CI/CD pipeline in place for testing.